receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -374,7 +377,10 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 		case writev2.Metadata_METRIC_TYPE_COUNTER:
 			addNumberDatapoints(metric.Sum().DataPoints(), ls, ts, &stats)
 		case writev2.Metadata_METRIC_TYPE_HISTOGRAM:
-			addHistogramDatapoints(metric.Histogram().DataPoints(), ls, ts)
+			// Histograms that comes with samples are considered as classic histograms and are not supported.
```

Comment:
Also : 
```
				sum.SetIsMonotonic(true)
				sum.SetAggregationTemporality(pmetric.AggregationTemporalityCumulative)
```

Reply:
[f037e05](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39864/commits/f037e0504b734df616f1143f590317c81c7eacbe)

Reply:
Turns out exponential histograms don't have monotonic flag, that was my mistake.